torchgeo/datasets/mapinwild.py

```diff
@@ -0,0 +1,513 @@
+        },
+        "s2_autumn": {
+            "https://huggingface.co/datasets/burakekim/mapinwild/resolve/main/s2_autumn/s2_autumn_part1.zip",  # noqa: E501
+            "https://huggingface.co/datasets/burakekim/mapinwild/resolve/main/s2_autumn/s2_autumn_part2.zip",  # noqa: E501
+        },
+        "s2_spring": {
+            "https://huggingface.co/datasets/burakekim/mapinwild/resolve/main/s2_spring/s2_spring_part1.zip",  # noqa: E501
+            "https://huggingface.co/datasets/burakekim/mapinwild/resolve/main/s2_spring/s2_spring_part2.zip",  # noqa: E501
+        },
+        "s2_summer": {
+            "https://huggingface.co/datasets/burakekim/mapinwild/resolve/main/s2_summer/s2_summer_part1.zip",  # noqa: E501
+            "https://huggingface.co/datasets/burakekim/mapinwild/resolve/main/s2_summer/s2_summer_part2.zip",  # noqa: E501
+        },
+        "s2_winter": {
+            "https://huggingface.co/datasets/burakekim/mapinwild/resolve/main/s2_winter/s2_winter_part1.zip",  # noqa: E501
+            "https://huggingface.co/datasets/burakekim/mapinwild/resolve/main/s2_winter/s2_winter_part2.zip",  # noqa: E501
+        },
+    }
+
+    split_url = "https://huggingface.co/datasets/burakekim/mapinwild/resolve/main/split_IDs/split_IDs.csv"  # noqa: E501
+
+    filenames = [
+        "ESA_WC.zip",
+        "VIIRS.zip",
+        "mask.zip",
+        "s1_part1.zip",
+        "s1_part2.zip",
+        "s2_autumn_part1.zip",
+        "s2_autumn_part2.zip",
+        "s2_spring_part1.zip",
+        "s2_spring_part2.zip",
+        "s2_summer_part1.zip",
+        "s2_summer_part2.zip",
+        "s2_temporal_subset_part1.zip",
+        "s2_temporal_subset_part2.zip",
+        "s2_winter_part1.zip",
+        "s2_winter_part2.zip",
+    ]
+
+    md5s = {
+        "ESA_WC.zip": "72b2ee578fe10f0df85bdb7f19311c92",
+        "VIIRS.zip": "4eff014bae127fe536f8a5f17d89ecb4",
+        "mask.zip": "87c83a23a73998ad60d448d240b66225",
+        "s1_part1.zip": "d8a911f5c76b50eb0760b8f0047e4674",
+        "s1_part2.zip": "a30369d17c62d2af5aa52a4189590e3c",
+        "s2_temporal_subset_part1.zip": "78c2d05514458a036fe133f1e2f11d2a",
+        "s2_temporal_subset_part2.zip": "076cd3bd00eb5b7f5d80c9e0a0de0275",
+        "s2_autumn_part1.zip": "6ee7d1ac44b5107e3663636269aecf68",
+        "s2_autumn_part2.zip": "4fc5e1d5c772421dba553722433ac3b9",
+        "s2_spring_part1.zip": "2a89687d8fafa7fc7f5e641bfa97d472",
+        "s2_spring_part2.zip": "5845dcae0ab3cdc174b7c41edd4283a9",
+        "s2_summer_part1.zip": "73ca8291d3f4fb7533636220a816bb71",
+        "s2_summer_part2.zip": "5b5816bbd32987619bf72cde5cacd032",
+        "s2_winter_part1.zip": "ca958f7cd98e37cb59d6f3877573ee6d",
+        "s2_winter_part2.zip": "e7aacb0806d6d619b6abc408e6b09fdc",
+    }
+
+    mask_palette = {1: (0, 153, 0), 0: (255, 255, 255)}
+
+    wc_palette = {
```

Comment:
We usually call these cmap instead of pallelete since that's what GDAL and matplotlib call them